tests/Service/ConfigurationFactoryTest.php

```diff
@@ -37,13 +39,21 @@ public function setUp(): void
     {
         $this->serviceManager = new ServiceManager();
         $this->factory        = new ConfigurationFactory('test_default');
-        $this->serviceManager->setService('doctrine.cache.array', new ArrayCache());
+        $this->serviceManager->setService('doctrine.cache.array', $this->getArrayCacheInstance());
         $this->serviceManager->setService(
             'doctrine.driver.orm_default',
             $this->createMock(MappingDriver::class)
         );
     }
 
+    protected function getArrayCacheInstance(): object
```

Comment:
Maybe this should be refactored to a trait so the logic can be shared between tests. I'm happy to do that if others think it's a good idea.